linetools/isgm/tests/files/group_9.VP

```diff
@@ -0,0 +1,4 @@
+specfile|restwave|zsys|col|sigcol|bval|sigbval|vel|sigvel|nflag|bflag|vflag|vlim1|vlim2|wobs1|wobs2|pix1|pix2|z_comp|trans 
```

Comment:
Optional columns "comment" and "reliability" may exist.

Reply:
same